### PR TITLE
UserGuide.adoc: Update command summary

### LINKED ISSUE #95: Update Command Summary for User Guide
Update UG for Command Summary part

### PR DESCRIPTION
**Pull request by Exeexe93**
Update all the commands Delino has in the commany summary.

### CODE REVIEW

**Review by khsc96 — APPROVED** [review]
Perhaps can include link to each section, but LGTM!

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -196,16 +196,25 @@ _{explain how the user can reschedule their orders when customers notify them th
...
-e.g. `import orders.csv`
-* *List* : `list`
+* *Import* : `import` `FILE_NAME` +
+e.g. `import` `orders.csv`
+* *List* : `list` `[DONE_STATUS]` +
+e.g. `list` `done`
+* *Return* : `return` `tid/TRANSACTION_ID` `n/NAME` `a/ADDRESS` `p/PHONE_NUMBER` `ts/DELIVERY_DATE_&_TIME` `w/WAREHOUSE_LOCATION` `pm/PAYMENT_METHOD` `c/COMMENTS_BY_CUSTOMER` `type/TYPE_OF_ITEM` +
+e.g. `return` `tid/ac17s2a` `n/BOBBY TAN` `a/123 Delta Road #03-333, Singapore 123456` `p/91230456` `ts/12-12-2020` `w/Jurong Warehouse` `pm/visa` `c/NIL` `type/glass`
+* *Sort* : `sort` `FLAG` `NAME/TIMESTAMP/TRANSACTION_ID` +
+e.g. `sort` `-d` `ts/`
```
Comment (JeremyLoh) [inline docs/UserGuide.adoc]: The sort command is based on any prefix that is related to the add command, not limited to name, timestamp and transaction id.
image

Comment (JeremyLoh) [reply docs/UserGuide.adoc]: I think can do
`sort` `[ORDER FLAG]` `ORDER_ATTRIBUTE_PREFIX`, then link to the sort section for more details

Comment (Exeexe93) [reply docs/UserGuide.adoc]: Resolved

**Review by JeremyLoh — APPROVED** [review]
Looks good to merge.

**Comment by Yuanyu97** [comment]
Good job on editing the user guide to fit your courier project!

**Pull request merged**